run_tests.py

```diff
@@ -56,13 +56,16 @@
         args.remove('--no-template')
         args.append('--ignore=tests')
         
+    # Set defautls for the test
```

Comment:
typo here

Reply:
I now realize this comment refers to deleted code. Removing